NightscoutUploadKit/Models/RemoteNotifications/BolusRemoteNotification.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  BolusRemoteNotification.swift
+//  NightscoutUploadKit
+//
+//  Created by Bill Gestrich on 2/25/23.
+//  Copyright © 2023 Pete Schwamb. All rights reserved.
+//
+
+import Foundation
+
+public struct BolusRemoteNotification: RemoteNotification, Codable {
+    
+    public let amount: Double
+    public let remoteAddress: String
+    public let expiration: Date?
+    public let sentAt: Date?
+    public let otp: String
+    
+    enum CodingKeys: String, CodingKey {
+        case remoteAddress = "remote-address"
```

Comment:
Is this encoding (`remote-address`) the Nightscout api encoding?  The json in the nightscout api typically uses camel case (`remoteAddress`).

Reply:
Yeah unfortunately the dash is used in the key. Here's the existing server and loop code that handles that 

https://github.com/nightscout/cgm-remote-monitor/blob/0bf616be873dd819b1842105dca14499d078dd5d/lib/server/loop.js#L60

https://github.com/LoopKit/Loop/blob/4909f52b7198582dab8b0b39697901594ef9311d/Loop/Models/RemoteCommand.swift#L16

Reply:
I had Taylor Swift's Anti-Hero song pop into my head when looking at the git blame for this. :P